src/Broker/Broker.c

```diff
@@ -26,21 +26,27 @@
 #include <stdint.h>
 
 #include <Packetizer.h>
+#include <ThinPacketizer.h>
+#include <DriverHelper.h>
 
 static Packetizer packetizer_data = { 0 };
+static enum PacketizerCfg selected_packetizer;
 
 __attribute__((section(".sdramMemorySection"))) static uint8_t packetizer_buffer[BROKER_BUFFER_SIZE];
 extern driver_send_function bus_to_driver_send_function[SYSTEM_BUSES_NUMBER];
 extern void* bus_to_driver_private_data[SYSTEM_BUSES_NUMBER];
 extern deliver_function interface_to_deliver_function[INTERFACE_MAX_ID];
+extern PacketizerFunctions packetizers[PACKETIZER_MAX_ID];
```

Comment:
If there are two buses using CCSDS, then there should be two CCSDS packetizers... right?